src/WebJobs.Extensions/Extensions/Timers/Scheduling/TimerSchedule.cs

```diff
@@ -98,5 +94,41 @@ internal static TimerSchedule Create(TimerTriggerAttribute attribute, INameResol
 
             return schedule;
         }
+
+        /// <summary>
+        /// For schedules with an occurrence more frequent than 1 minute, we disable schedule monitoring.
+        /// </summary>
+        /// <param name="cronSchedule">The cron schedule to check.</param>
+        /// <returns>True if monitoring should be disabled, false otherwise.</returns>
+        internal static bool ShouldDisableScheduleMonitor(CronSchedule cronSchedule, DateTime now)
+        {
+            // take the original expression minus the seconds portion
+            string expression = cronSchedule.InnerSchedule.ToString();
+            var expressions = expression.Split(' ');
+
+            // If any of the minute or higher fields contain non-wildcard expressions
+            // the schedule can be longer than 1 minute. I.e. the only way for all occurrences
+            // to be less than or equal to a minute is if all these fields are wild ("* * * * *").
+            bool hasNonSecondRestrictions = !expressions.Skip(1).All(p => p == "*");
+
```

Comment:
To clear layout the expectation of this behavior change. This would start to enable schedule monitor for many of timer trigger that we used not to. For instance, cron: * 0 20 * * *,  since scale controller would only warm up the site a minute and half before, in the old behavior, due to the race condition,  we would not enable schedule monitor even we know each batch of operation is 24 hr apart. After this change, even each execution within one batch is every second, we would still enable schedule monitor.  This should not be too big of stress for the host or storage handle, but if we think about an worse case like * *20 * * *,  this might result so billing cost on the storage, but we can tell customer to explicit opt out. I am cool with overall approach but just point out the potential risk, thoughts?

Reply:
Yes, it's a bug fix, so cases where we weren't enabling but should have been, we'll now enable, which may improve reliability for previously affected apps.

Regarding storage transactions - yes, if you've defined a schedule that can run for a period of consecutive seconds, that will result in a storage update per invocation. But that's nothing storage can't handle, and operations are very inexpensive.